PhysicsTools/NanoAOD/python/simpleSingletonCandidateFlatTableProducer_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+from PhysicsTools.NanoAOD.simpleCandidateFlatTableProducer_cfi import simpleCandidateFlatTableProducer as default
+
+simpleSingletonCandidateFlatTableProducer = default.clone(
```

Comment:
why is this needed?

Reply:
the default cgi is for `singleton=false `that requires `cut` to be specified. cloning with `singleton=True ` would pul in `cut=""` and a parameter validation error. `simpleCandidateFlatTableProducer_cfi` makes it proper default parameter (i.e. without `cut` defined at all)